src/approaches/oracle_approach.py

```diff
@@ -1001,3 +1010,183 @@ def _get_repeated_nextto_gt_nsrts() -> Set[NSRT]:
     nsrts.add(grasp_nsrt)
 
     return nsrts
+
+
+def _get_behavior_gt_nsrts() -> Set[NSRT]: # pragma: no cover
+    """Create ground truth nsrts for BehaviorEnv."""
+    env = get_env_instance("behavior")
+
+    type_name_to_type = {t.name: t for t in env.types}
+    pred_name_to_pred = {p.name: p for p in env.predicates}
+
+    def _get_lifted_atom(
```

Comment:
suggest refactor to use `_get_predicate` below inside, to remove redundant lines

Reply:
Sorry, not sure what you mean by this? Could you elaborate/give an example?

Reply:
sure sorry -- I meant:

```
    def _get_lifted_atom(        
        base_pred_name: str, objects: Sequence[Variable]
    ) -> LiftedAtom:
        pred = _get_predicate(base_pred_name, [o.type for o in objects])
        return LiftedAtom(pred, objects)
```